invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html

```diff
@@ -10,7 +10,7 @@
 #}
 
 
-{%- macro preview_file(preview_endpoint, pid_value, filename, is_preview, include_deleted, id='preview-iframe', width='100%', height='400' ) %}
+{%- macro preview_file(preview_endpoint, pid_value, filename, is_preview, include_deleted, id='preview-iframe', width='100%', height='800' ) %}
```

Comment:
we are changing the default `height` for all previewers here, not only Mirador. Is this what we want?

Reply:
400px is pretty small, so making it bigger for everyone might be something desirable. See e.g. <https://new-cds.cern.ch/records/fp18d-jc149>:

<img width="1127" alt="image" src="https://github.com/inveniosoftware/invenio-app-rdm/assets/1814661/267185d7-a1e7-4a50-b9ca-74a4881fe5b4">

We have to be a bit careful on mobile though, it's a bit difficult to touch/scroll at the right place to get past the previewer:

<img width="458" alt="image" src="https://github.com/inveniosoftware/invenio-app-rdm/assets/1814661/490f3b61-cd99-4e8b-8a83-3d74ded64a75">

We should probably rewrite to use CSS classes for styling, with `80vh` or similar (that way anyone can override also on their instance).